src/components/API.js

```diff
@@ -0,0 +1,172 @@
+// b73a4638-eb7a-419c-8b34-f34f4d770b05 auth token
+
+export default class Api {
+  constructor(key, url) {
+    this._key = key;
+    this._url = url;
+  }
+
+  getPromiseAll(funcs) {
+    return Promise.all(funcs)
+  }
+
+  // Profile
+
+  getUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
+      headers: {
+        authorization: this._key
+      }
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)